shared/test/dataDriver_test.js

```diff
@@ -21,6 +21,10 @@ describe("loop.DataDriver", () => {
     driver = new loop.DataDriver({
       dispatcher
     });
+    let originalConnect = driver._connectToRoom;
+    sandbox.stub(driver, "_connectToRoom", token => {
+      originalConnect.call(driver, token);
+    });
```

Comment:
Any reason why you're stubbing this for all tests and not just for the one test? Typically you stub things so that the original function doesn't get called.